restaurants/views.py

```diff
@@ -1,3 +1,53 @@
 from django.shortcuts import render
 
-# Create your views here.
+from django.views         import View
+from django.http          import JsonResponse
+from django.db.models     import Avg
+
+from restaurants.models   import Restaurant, Category, Location, ServingPrice, Menu
+from users.models         import User, WishList, Rating
+from reviews.models       import Review, ReviewImage
+from users.utils          import login_decorator
+
+class RestaurantDetailView(View):
+    def get(self, request, restaurant_id):
+        try:
+            restaurant         = Restaurant.objects.get(id = restaurant_id)
+            reviews            = Review.objects.filter(restaurant_id = restaurant_id)
```

Comment:
```suggestion
    # 1안)
            restaurant         = Restaurant.objects.get(id = restaurant_id)
            reviews            = Review.objects.filter(restaurant_id = restaurant_id)
           
    # 2안)
            restaurant         = Restaurant.objects.get(id = restaurant_id)
            restaurant.reviews_set.all()
```

1안과 2안을 테스트해보시고, 어떠한 차이가 있는지 확인해주세요!
이 부분 파악하시면 아래 코드도 동일하게 반영가능합니다!

Reply:
2안이 불필요한 주고받음을 줄일 수 있을 것 같습니다~